aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -849,12 +851,22 @@ class AztecToolbar : FrameLayout, OnMenuItemClickListener {
         headingMenu = PopupMenu(context, view)
         headingMenu?.setOnMenuItemClickListener(this)
         headingMenu?.inflate(R.menu.heading)
+        headingMenu?.setOnDismissListener({
+            if (getSelectedHeadingMenuItem() == null || getSelectedHeadingMenuItem() == AztecTextFormat.FORMAT_PARAGRAPH) {
```

Comment:
I believe we should add an 'else' branch with `isChecked = true`. WDYT?
Issue
1. Select H1 heading
2. Open heading pop-up menu
3. Without selecting anything press back button and close the pop-up menu.
4. Focus on the Heading button - talkback announces `not checked`